ac

```diff
@@ -0,0 +1,796 @@
+            . $VENV_BIN/activate && cd ~/.ansible/collections/ansible_collections/ibm/ibm_zos_core && \
+                 ${VENV_BIN}/ansible-test sanity --python ${option_version} --requirements --docker default && \
+                    cd ${CURR_DIR};
+        else
+            message "Running ansible-test with docker container and all python versions."
+            . $VENV_BIN/activate && cd ~/.ansible/collections/ansible_collections/ibm/ibm_zos_core && \
+                ${VENV_BIN}/ansible-test sanity --requirements --docker default && \
+                    cd ${CURR_DIR};
+        fi
+    else
+        if [ "${option_version}" ]; then
+            message "Docker engine is not running, version ${option_version} will be ignored."
+        fi
+
+        . $VENV_BIN/activate && VENV_PY_VER=`python3 --version | cut -d" " -f2 | cut -d"." -f1,2`
+        message "Running ansible-test with managed python virtual environment: ${VENV}."
+		. $VENV_BIN/activate && cd ~/.ansible/collections/ansible_collections/ibm/ibm_zos_core && \
+			${VENV_BIN}/ansible-test sanity --python ${VENV_PY_VER} --requirements && \
+				cd ${CURR_DIR};
+    fi
+}
+
+# ------------------------------------------------------------------------------
+# Run functional tests:
+# ------------------------------------------------------------------------------
+#->ac-test:
+## Run the functional tests inside the managed python virtual environment.
+## Usage: ac [--ac-test] [--host <host>] [--python <python>] [--zoau <zoau>] [--file <file>] [--debug <boolean>]
+## Options:
+##     host     - z/OS managed node to run test cases, no selection defaults to
+##                a host registerd to your user id (`whoami`).
+##     python   - IBM enterprise python version, choices are 3.8, 3.9, 3.10, 3.11,
+##                no selection defauls to 3.8.
+##     zoau     - ZOAU to use in testing, choices are 1.0.3, 1.1.1, 1.2.0, 1.2.1,
+##                no selection defaults to 1.1.1 .
+##     file     - the absoulte path to a test suite to run, no selection
+##                defaults to all tests running.
+##     debug    - enable debug for pytest (-s), choices are true and false
+## Example:
+##  $ ac --ac-test --host ec33012a --python 3.10 --zoau 1.2.2 --file tests/functional/modules/test_zos_operator_func.py --debug true
+##  $ ac --ac-test --file tests/functional/modules/test_zos_operator_func.py --debug true
+##  $ ac --ac-test
+ac_test(){
+    host=$1
+    python=$2
+    zoau=$3
+    file=$4
+    debug=$5
+    skip=$CURR_DIR/tests/functional/modules/test_module_security.py
+
+    # Create the config always overwriting existing
+    ${VENV}/./venv.sh --config ${host} ${python} ${zoau} ${VENV}
+
+	# Check configuration was created in venv/config.yml, else error and exit
+	if test ! -e  ${VENV}/config.yml; then
+	    echo "No configuration was able to be created in ${VENV}/config.yml "
+		exit 1
+	fi
+
+    cd ${VENV_BIN}
```

Comment:
Is this `cd` needed? The paths used to setup and run tests are absolute paths. I had problems with `pytest` not picking up my `ansible.cfg` inside the repo root because the working directory is `VENV_BIN`.

Reply:
I don't think so, i commented them out, I think they were there before I set up the absolute paths, good catch.